lib/surface/api.ex

```diff
@@ -365,16 +369,6 @@ defmodule Surface.API do
 
   defp maybe_warn_mutually_exclusive_opts(_, _, _, _), do: nil
 
-  defp validate_required_opts(func, type, opts) do
```

Comment:
This function will always return `:ok` as the `get_required_opts` always return `[]`. Moreover, I haven't found any references to required opts in the code.

Reply:
Also leftover from the old `context` API :)